src/CommonLib/aspnetcoreconfig.h

```diff
@@ -215,18 +215,28 @@ class ASPNETCORE_CONFIG : IHttpStoredContext
         return &m_struConfigPath;
     }
 
+    PCWSTR**
+    QueryHostFxrArguments(
+        VOID
+    )
+    {
+        return &m_pcwArguments;
+    }
+
     STRU*
-    QueryHostfxrPath()
+    QueryHostFxrLocation(
+        VOID
+    )
     {
-        return &m_struHostFxrPath;
+        return &m_struHostFxrLocation;
     }
 
-    BOOL
-    QueryIsStandAloneApplication(
+    DWORD*
+    QueryHostFxrArgCount(
```

Comment:
theoretically this counter can be out of sync with the string array. Maybe you can try dynamically calculate the value sizeof(m_pcwArguments)/sizeof(m_pcwArguments[0]) with null check